slack_etiquette.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Slack Etiquette
+permalink: /about/slacketiquette
+published: true
+---
+
+
+# Slack Etiquette
+### Aka meming and messaging responsibly - msoe-sse.slack.com
+> Few people want constant phone buzzes 
+
+## 1. Unless you are on the executive board, do NOT use @channel in #general or #random under any circumstances
+> If you do have something of importance, please message _Brady Cassada_, the Communications Coordinator
+
+## 2. Threads are your friend! They don't clog up the channel, and they keep conversation focused. 
+> In your **thread reply**, be sure to **@whoever** you're replying to in the thread otherwise they might not get a notification!
+
+## 3. If you are creating a channel for a small specific group (for example, class project groups), 
+> **Make it private.** 
+> Or, if the channel is around a particular interest, idea, or topic that you'd like others to get involved in, then consider making it public.

```

Comment:
Here too, a screenshot might be helpful. It might be obvious but to a newcomer it might be helpful--remember that it's largely newcomers who'd be reading this in the first place :)